docs/source/upcoming_release_notes/302-repair_match_name_to_id.rst

```diff
@@ -0,0 +1,22 @@
+302 repair_match_name_to_id
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Bugfixes
+--------
+- Issue 302: Add functionality to happi 'repair' that ensures that the name and id fields of a device are the same. 
```

Comment:
See our `pre-commit` output here in the checks? https://github.com/pcdshub/happi/actions/runs/5942583794/job/16115855304
It's verifying some basic style stuff and it found an issue on this line I'm commenting on - turns out you have a bit of whitespace at the end of the line.

`pre-commit` will actually fix this automatically for you, if you run it locally.
Now, you can certainly run pre-commit stuff on-demand, but I find it helpful to fully install it (by way of `pre-commit install`) so it runs each time before you've made a commit.

Our hooks are defined at the repo top level in `.pre-commit-config.yaml`
Here's a link to the usage: https://pre-commit.com/#usage and if you scroll up to the top, there's some useful background information there, too.

Reply:
I think this is the only active problem with this PR